src/pages/RepoPage/FlagsTab/subroute/FlagsTable/FlagsTable.tsx

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable max-nested-callbacks */
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  OnChangeFn,
+  SortingState,
+  useReactTable,
+} from '@tanstack/react-table'
+import cs from 'classnames'
+import { memo, useEffect, useMemo, useState } from 'react'
+import { useInView } from 'react-intersection-observer'
+import { useParams } from 'react-router-dom'
+
+import { useRepoConfig } from 'services/repo/useRepoConfig'
+import { determineProgressColor } from 'shared/utils/determineProgressColor'
+import A from 'ui/A'
+import CoverageProgress from 'ui/CoverageProgress'
+import Icon from 'ui/Icon'
+import Spinner from 'ui/Spinner'
+
+import DeleteFlagModal from './DeleteFlagModal'
+import useRepoFlagsTable from './hooks'
+import TableSparkline from './TableEntries/TableSparkline'
+
+import 'ui/Table/Table.css'
+
+interface FlagsTableHelper {
+  name: JSX.Element
+  coverage: JSX.Element
+  trend: JSX.Element
+  delete: JSX.Element | null
```

Comment:
Can you swap these to `React.ReactNode` as `JSX.Element` does not allow valid returns such as `null`, `number`, etc.